src/devices/MettlerToledo/Messages.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Iot.Device.MettlerToledo.Messages
+{
+    internal static class Errors
+    {
+        /// <summary>
+        /// The balance has not recognized the received command.
+        /// </summary>
+        internal const string SYNTAX_ERROR = "ES";
```

Comment:
Please adjust the constant to be compliant with the naming convention. it should be `SyntaxError`. The exception are for register to match the documentation. In the documentation, the only names are the ES, ET, etc, so the naming convention should apply everywhere.